backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/UpdateVmCommand.java

```diff
@@ -1394,7 +1394,8 @@ && isHotSetEnabled()
             return failValidation(EngineMessage.ERROR_CANNOT_FIND_ISO_IMAGE_PATH);
         }
         if (!validate(vmHandler.validateCpuPinningPolicy(getParameters().getVmStaticData(),
-                getParameters().isUpdateNuma(), getEffectiveCompatibilityVersion()))) {
+                getVm().getStaticData(),
```

Comment:
that's the VM from the database (with the previous configuration), no?

Reply:
Yes, we want to compare the vm from the parameters with the vm from the database to see if the values have been changed.

Reply:
ah okok, we get to the second validateCpuPinningPolicy below